packages/hadron-express/src/hadronToExpress.ts

```diff
@@ -38,7 +38,18 @@ const createRoutes = (
 
             return newRouteCallback(request, containerProxy);
           })
-          .then(handleResponseSpec(res))
+          .then((cb) => {
```

Comment:
Type would be usefull. Also let's try to avoid abbreviation of variables name (`cb`). It reduces readability without any real benefits.